e2e/libs/backing_image/backing_image.py

```diff
@@ -126,3 +129,18 @@ def wait_for_backing_image_manager_on_node_terminated(self, node_name):
 
     def get_backing_image_disk_uuids(self, bi_name):
         return self.backing_image.get_backing_image_disk_uuids(bi_name)
+
+    def download_backing_image(self, bi_name):
+        if os.getenv('LONGHORN_CLIENT_URL'):
+            longhorn_client_url = os.getenv('LONGHORN_CLIENT_URL')
+            cmd = f'curl -fL "{longhorn_client_url}/v1/backingimages/{bi_name}/download" | gunzip -c > /tmp/{bi_name}'
+            subprocess_exec_cmd(cmd)
+            cmd = f"sha512sum /tmp/{bi_name} | cut -d' ' -f1"
+            res = subprocess_exec_cmd(cmd)
+            return res.strip()
+        else:
+            assert False, "Can not get environment variable LONGHORN_CLIENT_URL"
```

Comment:
Does that mean this test case never works on Jenkins pipeline? Why can't it get manager ips just like how robot/pytest initializes Longhorn client?

Reply:
Hi @yangchiu , updated, thank you. https://10.115.5.5/job/private/job/longhorn-e2e-test/127/